sigle/src/modules/publicHome/components/PublicHome.tsx

```diff
@@ -71,6 +88,7 @@ interface PublicHomeProps {
 export const PublicHome = ({ file, settings }: PublicHomeProps) => {
   const router = useRouter();
   const { username } = router.query as { username: string };
+  const { data: stacksApiNameInfo } = useGetStacksApiNameInfo(username);
```

Comment:
@gregogun we now inject the address directly from the props, can you please remove the hook and just use the props? https://github.com/sigle/sigle/pull/568